qml/pages/FirstPage.qml

```diff
@@ -79,13 +122,18 @@ Page {
                     }
                 }
             }
+//            MenuItem {
+//                enabled: gpsDataSource.active
+//                text: qsTr("Share Location")
+//                onClicked: pageStack.push(Qt.resolvedUrl("SharePage.qml"))
+//            }
```

Comment:
Can you please push SharePage.qml too (in its current state), altough it's commented out?

Reply:
I think I will remove it, unless I make some progress on it this week.  
I thought I have a working shargepage.qml last year, but seems to have gone.